frontend/src/components/landingPage/MenuStepTwo.vue

```diff
@@ -11,7 +11,7 @@
 					<option>=</option>
 				</select>
 				<input class="form-control" id="valueInput" type="text" placeholder="Input Value" v-model="selectedValue" @input="sendValue">  
-				<h6 id="nanInformation" v-show="notANumber">Value should be numerical</h6>           
+				<h6 id="nanInformation" v-show="notANumber">Number Please</h6>           

```

Comment:
I liked the former text better... much more polite. :)

Reply:
I changed it back. Also decreased the font size a little bit.